parts/k8s/cloud-init/artifacts/cse_main.sh

```diff
@@ -179,6 +179,7 @@ ps auxfww > /opt/azure/provision-ps.log &
 
 if $FULL_INSTALL_REQUIRED; then
   applyCIS || exit $ERR_CIS_HARDENING_ERROR
+  sysctl_reload 20 5 10 || exit $ERR_SYSCTL_RELOAD
```

Comment:
@CecileRobertMichon This is how we enable these changes for non-VHD scenario

Reply:
Cool thanks. How about running that in the VHD build? Or will VMs starting from that VHD automatically pick up the configs on the disk?

Reply:
We don't need it in the VHD build process unless we want to run validation tests, because the configuration will be applied at next boot.

Reply:
Nevermind, we don't have a VHD implementation for these cloud-init-delivered changes, hmm...

Reply:
yeah that was what I meant